packages/venia-ui/lib/components/AccountMenu/__tests__/__snapshots__/accountMenuItems.spec.js.snap

```diff
@@ -2,13 +2,13 @@
 
 exports[`it renders correctly 1`] = `
 <div>
-  &lt;Link&gt;Order History&lt;/Link&gt;
-  &lt;Link&gt;Store Credit & Gift Cards&lt;/Link&gt;
-  &lt;Link&gt;Favorites Lists&lt;/Link&gt;
-  &lt;Link&gt;Address Book&lt;/Link&gt;
-  &lt;Link&gt;Saved Payments&lt;/Link&gt;
-  &lt;Link&gt;Communications&lt;/Link&gt;
-  &lt;Link&gt;Account Information&lt;/Link&gt;
+  &lt;Link&gt;[object Object]&lt;/Link&gt;
+  &lt;Link&gt;[object Object]&lt;/Link&gt;
+  &lt;Link&gt;[object Object]&lt;/Link&gt;
+  &lt;Link&gt;[object Object]&lt;/Link&gt;
+  &lt;Link&gt;[object Object]&lt;/Link&gt;
+  &lt;Link&gt;[object Object]&lt;/Link&gt;
+  &lt;Link&gt;[object Object]&lt;/Link&gt;
```

Comment:
Is this change intentional? Should we not still have a string represented in the snapshot?

Reply:
I noticed this. I do think we need to mock something in order for snapshots to be accurate. The text replacement logic in `react-intl` components seems pretty robust, so this isn't too surprising.